galaxykit/command.py

```diff
@@ -300,6 +300,8 @@ def main():
                 print_unknown_error(args)
 
         elif args.kind == "collection":
+            if args.operation == "list":
+                print(json.dumps(collections.get_collection_list(client)))
             if args.operation == "upload":
```

Comment:
`elsif`

otherwise if works but ends up with `Unknown collection operation 'list'`

Reply:
Hm, yes, thanks, somehow this error did not came up at testing it out. I need to learn that this is not a switch statement. :face_with_spiral_eyes: